src/configurations/nf/routesConfig.ts

```diff
@@ -48,6 +48,12 @@ const routes: GenericRoute[] = [
     path: '',
     exact: true,
```

Comment:
Could probably put the component in a route with `path: ''` and `exact: false` if you wanted it to appear on any page in the portal.

Reply:
Great idea!